tests/conftest.py

```diff
@@ -17,6 +19,16 @@ def mock_execution_context(mocker):
     return mock_context
 
 
+@pytest.fixture
+def mock_locations(mocker):
+    locations = mocker.patch("cascade.input_data.db.locations.location_hierarchy")
+    G = nx.DiGraph()
+    G.add_nodes_from(list(range(8)))
+    G.add_edges_from([(0, 1), (0, 2), (1, 3), (1, 4), (2, 5), (2, 6), (6, 7)])
```

Comment:
nice.

Reply:
Pretty sure that's your mock. I just moved it.